reference-implementation/test/writable-stream.js

```diff
@@ -473,154 +473,6 @@ test('If close is called on a WritableStream in waiting state, ready will be ful
   }, 0);
 });
 
-test('If sink rejects on a WritableStream in writable state, ready will return a fulfilled promise', t => {
```

Comment:
Hmm, I guess it might be worth porting this test more directly as well. Will do.